Base/res/devel/templates/cpp-basic.ini

```diff
@@ -0,0 +1,4 @@
+[HSTemplate]
```

Comment:
Perhaps we could just name this `HackStudioTemplate`? Doesn't seem *too* long to me, and less ambiguous.